Dockerfile.kubectl

```diff
@@ -0,0 +1,12 @@
+FROM alpine:3.17.3
```

Comment:
This image doesn't have bash, and I think we want to have it even if it's not strictly necessary for the pvcCleaner script.

How about using the [official bash image](https://hub.docker.com/_/bash) as a base? That's also based on alpine, and scans come out clean on it.

Reply:
So do you propose to use `bash` instead of `scratch` or instead of `alpine` in this Dockerfile?